backend/src/config/matomo.rs

```diff
@@ -7,13 +7,23 @@ use crate::prelude::*;
 pub(crate) struct MatomoConfig {
     /// URL of your Matomo server. Example: "https://matomo.myuni.edu/matomo/".
     ///
-    /// Note: Adding `matomo.js` to the URL configured here should result in a
-    /// URL to a publicly accessible JS file.
+    /// Note: Adding the filename of the Matomo script to the URL configured here should result in
+    /// a URL to a publicly accessible JS file.
     #[config(deserialize_with = deserialize_server)]
     pub(crate) server: Option<Uri>,
 
     /// Matomo site ID, e.g. `side_id = "1"`
     pub(crate) site_id: Option<String>,
+
+    /// Filename for the Matomo JS script.
+    /// Default: "matomo.js"
```

Comment:
You don't have to specify "Default" yourself, the library already does that. Take a look at the generated config file